latest-thread.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Latest LOOT thread
+permalink: /latest-thread/
+redirect_to:
+  - http://forums.bethsoft.com/topic/1588424-rel-loot-thread-22/
+---
+
+<p>The latest thread about LOOT on the Bethesda forums can be found at:
```

Comment:
Ah. Hm. I've commented about this on https://github.com/jekyll/jekyll-redirect-from/issues/55 - I'm fine with ripping those parts (`layout` and content) out and adding them back in if it ever becomes relevant, but OTOH, leaving them in would still provide a fall-back in case the plugin/extension get disabled for whatever reason (doesn't get updated for a jekyll update, mysteriously disappears due to GitHub internals, …).

Either way, feel free to make an update commit or tell me what you'd like to see done and I'll poke at it. :)